packages/app/src/app/pages/NewDashboard/Content/routes/Settings/TeamSettings.tsx

```diff
@@ -380,3 +441,21 @@ export const TeamSettings = () => {
     </>
   );
 };
+
+const getAuthorization = (user, team) => {
+  let authorization;
+
+  if (team.userAuthorizations) {
+    authorization = team.userAuthorizations.find(
+      auth => auth.userId === user.id
+    ).authorization;
+  } else if (user.id === team.creatorId) {
+    // backward compatible
+    authorization = 'ADMIN';
+  } else {
+    // backward compatible
+    authorization = 'WRITE';
+  }
+
+  return authorization;
+};
```

Comment:
This will make the function a bit clearer imo

```suggestion
const getAuthorization = (user, team) => {
  if (team.userAuthorizations) {
    return team.userAuthorizations.find(
      auth => auth.userId === user.id
    ).authorization;
  }
  
  if (user.id === team.creatorId) {
    // backward compatible
    return 'ADMIN';
  }
  
  // backward compatible
  return 'WRITE';
};
```